src/authz/ldap.js

```diff
@@ -28,46 +27,27 @@ class LDAPAuthorizer {
     assert(options.cfg.ldap.password, 'options.cfg.ldap.password is required');
     assert(options.cfg.ldap.allowedGroups, 'options.cfg.ldap.allowedGroups is required');
 
+    this.user = options.cfg.ldap.user;
+    this.password = options.cfg.ldap.password;
+    this.client = new LDAPClient(options.cfg.ldap);
```

Comment:
Again I would suggest wrapping in a Synchronize contruct again...

Maybe even use the same LDAPClient object, to reduce number of TCP connections that can break...